README.md

```diff
@@ -13,7 +13,7 @@ HTTP and HTTPS listeners with default actions:
 ```hcl
 module "alb" {
   source  = "terraform-aws-modules/alb/aws"
-  version = "~> 6.0"
+  version = "~> 8.1.0"
```

Comment:
Could you please change it to be "~> 8.0" everywhere in the readme? LGTM otherwise :)